File: Daily/1897. Redistribute Characters to Make All Strings Equal.cpp
class Solution {
public:
    bool makeEqual(vector<string>& words) {
        int n = words.size();
        unordered_map<char, int> mp;
        for(int i = 0; i < n; i++) {
            for(int j = 0; j < words[i].size(); j++) {
                mp[words[i][j]]++;
            }
        }
        for(auto it = mp.begin(); it != mp.end(); it++) {
            if(it->second % n != 0) return false;
        }
        return true;      
    }
};

File: Daily/1897. Redistribute Characters to Make All Strings Equal.js
/**
 * @param {string[]} words
 * @return {boolean}
 */
var makeEqual = function(words) {
    let map = new Map();
    for (let word of words) {
        for (let c of word) {
            map.set(c, (map.get(c) || 0) + 1);
        }
    }
    for (let v of map.values()) {
        if (v % words.length !== 0) {
            return false;
        }
    }
    return true;
};